addon/components/document-view.js

```diff
@@ -15,7 +15,7 @@ export default class DocumentViewComponent extends Component {
 
   @tracked isDragOver = false;
   @tracked dragCounter = 0;
-  @tracked listView = true;
+  @tracked listView = this.args.listView === "true";
```

Comment:
Can't you just always use `this.args.listView` and eliminate the tracked property entirely?